code/game/objects/items/melee/baton.dm

```diff
@@ -632,7 +629,7 @@
 
 /obj/item/melee/baton/security/throw_impact(atom/hit_atom, datum/thrownthing/throwingdatum)
 	. = ..()
-	if(active && prob(throw_stun_chance) && isliving(hit_atom))
+	if(!. && active && prob(throw_stun_chance) && isliving(hit_atom))
```

Comment:
i have a sneaking suspicion that this will be easier to read as an early return using `||`, but i could just be messed up in the head